lambda/update/main_test.go

```diff
@@ -27,16 +27,25 @@ func (m *mockStore) Put(ctx context.Context, data any) error {
 	m.put = data
 	return m.putErr
 }
+func (m *mockStore) PutChanges(ctx context.Context, data any, update shared.Update) error {
+	m.put = data
```

Comment:
We should capture the `update` data here and assert it in the tests to ensure it's capturing the right data.
```suggestion
	m.update = update
```